docker/env/hydra.sh

```diff
@@ -25,8 +26,32 @@ SCT_ARGUMENTS=()
 
 while [[ $# -gt 0 ]]; do
     case $1 in
+        --execute-on-new-runner)
```

Comment:
It is argument parsing logic, not a best place to run things.
Please move this logic at right after both while cycles

Reply:
Fixed